src/test/ui/inline-asm-bad-operand.stderr

```diff
@@ -37,8 +37,8 @@ LL |         asm!("mov sp, $0"::"r"(addr),
 error[E0669]: invalid value for constraint in inline assembly
   --> $DIR/inline-asm-bad-operand.rs:56:32
    |
-LL |                            "r"("hello e0669"));
-   |                                ^^^^^^^^^^^^^
+LL | ...                   "r"("hello e0669"));
```

Comment:
should we trim whitespace more eagerly?

Reply:
Maybe, although I want it always to be clear that trimming is happening (with the leading `...` and some leading whitespace as possible).